### PR TITLE
Fixes washing machine

### LINKED ISSUE #14145: Throwing TK moves you in no-gravity
In no-gravity, you can telekinesis an item, and throw the telekinesis to move in no-gravity.

### LINKED ISSUE #17462: Washing Machines no longer allow items to be dyed
Ever since we had the icon glitch several days or so ago, none of the dyeable items (such as bed sheets and shoes) have been appearing as dyed when you wash them with crayons. 
Can be recreated by simply dropping a pair of shoes and crayon into a washer and turning it on.

I need muh fashions

### PR DESCRIPTION
**Pull request by phil235**
- Moved crayon code out of toys.dm and into crayons.dm because why would you not, and also toys.dm is already huge (1000+ lines).
- Fixes being able to throw kinetic grabs. Fixes #14145
- Fixes washing machine code being shit. Also fixes clothes coloring via washing machine being broken. Fixes #17462 . And fixes Head stamp not being able to make a colored Head jumpsuit. 
- Fixes bloodied washing machine being uncleanable making it forever unusable.

### CODE REVIEW

**Comment by KazeEspada** [comment]
Washing machine code fixed... THE END IS NEAR

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -341,7 +341,6 @@ BLIND     // can't see anything
 	var/can_adjust = 1
 	var/adjusted = 0
 	var/alt_covers_chest = 0 // for adjusted/rolled-down jumpsuits, 0 = exposes chest and arms, 1 = exposes arms only
-	var/suit_color = null
```
Comment (phil235) [inline code/modules/clothing/clothing.dm]: good ol paprika, adding a var to all jumpsuit to resolve one fucking issue that he created. Good job... All that was needed was two lines in update_inv_w_uniform().

File: code/modules/clothing/shoes/colour.dm [hunk 2]
```diff
@@ -11,27 +11,32 @@
...
-		item_color = "captain"	//Exists for washing machines. Is not different from brown shoes in any way.
-	hop
-		item_color = "hop"		//Exists for washing machines. Is not different from brown shoes in any way.
-	ce
-		item_color = "chief"		//Exists for washing machines. Is not different from brown shoes in any way.
-	rd
-		item_color = "director"	//Exists for washing machines. Is not different from brown shoes in any way.
-	cmo
-		item_color = "medical"	//Exists for washing machines. Is not different from brown shoes in any way.
-	cmo
```
Comment (phil235) [inline code/modules/clothing/shoes/colour.dm]: ngggggggg

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -324,223 +324,6 @@
...
-	var/paint_color = "#FF0000" //RGB
-	var/drawtype = "rune"
-	var/text_buffer = ""
-	var/list/graffiti = list("amyjon","face","matt","revolution","engie","guy","end","dwarf","uboa","body","cyka","arrow","star","poseur tag")
-	var/list/letters = list("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z")
-	var/list/numerals = list("0","1","2","3","4","5","6","7","8","9")
-	var/list/oriented = list("arrow","body") // These turn to face the same way as the drawer
-	var/uses = 30 //-1 or less for unlimited uses
-	var/instant = 0
-	var/colourName = "red" //for updateIcon purposes
```
Comment (phil235) [inline code/game/objects/items/toys.dm]: that var has exactly the same function and values as item_color. I removed it.

File: code/game/machinery/washing_machine.dm [hunk 4]
```diff
@@ -26,286 +20,227 @@
...
-			return
-		if ( istype(W,/obj/item/clothing/mask/cigarette ) )
-			user << "This item does not fit."
-			return
-		if ( istype(W,/obj/item/clothing/head/syndicatefake ) )
-			user << "This item does not fit."
-			return
-		if ( istype(W,/obj/item/clothing/head/helmet ) )
-			user << "This item does not fit."
-			return
```
Comment (phil235) [inline code/game/machinery/washing_machine.dm]: quality code right there

File: code/game/machinery/washing_machine.dm [hunk 5]
```diff
@@ -26,286 +20,227 @@
...
+				break
+
+
+/obj/machinery/washing_machine/relaymove(mob/user)
+	container_resist(user)
+
+/obj/machinery/washing_machine/container_resist(mob/user)
+	if(!busy)
+		add_fingerprint(user)
+		open_machine()
```
Comment (phil235) [inline code/game/machinery/washing_machine.dm]: the player controlled corgi can escape the killing washing machine now.

File: code/game/machinery/washing_machine.dm [hunk 6]
```diff
@@ -26,286 +20,227 @@
...
+	overlays.Cut()
+	if(busy)
+		icon_state = "wm_running_[bloody_mess]"
+	else if(bloody_mess)
+		icon_state = "wm_[state_open]_blood"
+	else
+		var/full = contents.len ? 1 : 0
+		icon_state = "wm_[state_open]_[full]"
+	if(panel_open)
+		overlays += image(icon, icon_state = "wm_panel")
```
Comment (phil235) [inline code/game/machinery/washing_machine.dm]: by using an overlay for the open panel, we divide by two the number of sprites needed.

**Comment by Jalleo** [comment]
Thank you for fixing the washing machine properly phil. There may be other issues somewhere that are closeable with this.

**Comment by PKPenguin321** [comment]
Holy fuck Phil fixed washing machines

Coder of the year

**Comment by Durkel** [comment]
Press F to pay respects to legendary shit code.

**Comment by Iamgoofball** [comment]
F
FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF

File: code/game/machinery/washing_machine.dm (hunk 4 above)
Comment (optimumtact) [inline code/game/machinery/washing_machine.dm]: this item does not fit

Comment (PKPenguin321) [inline code/game/machinery/washing_machine.dm]: jesus fuckign christ isf this dude is so allergic to a new var on every clothes item the least he could have done was made these one check with a gorillion ||s
what the fuck how could he possibly have decided that this was the best solution

**Comment by Cheridan** [comment]
what's the next worst code in the repo?

**Comment by optimumtact** [comment]
@Cheridan it's you :laughing:

**Pull request merged**